### DISCUSSION TITLE
zmap 7.0

### DISCUSSION THREAD

**Discussion by ryuki1234** [zmap/zmap#866] [Q&A] [answered]
![Screenshot (410)](https://github.com/zmap/zmap/assets/168662421/47010417-df90-4958-a51b-2216698643fa)
how to fix error like this?

i need help, because very urgent

**Comment by droe** [comment, marked as answer]
I think you are looking for a different ZMap, the Seismology MATLAB library of the same name lives over at https://github.com/CelsoReyes/zmap7
